spec/steps/discover.fmf

```diff
@@ -1,44 +1,99 @@
-summary: Discover test relevant for execution
+summary: Discover tests relevant for execution
 
 description: |
-    Gather information about the test cases which are supposed to
-    be run. This includes list of test cases with corresponding
-    L1 metadata. From the test case metadata constraints for test
-    enviroment can be detected:
+    Gather information about tests which are supposed to be run.
+    Provide method ``tests()`` returning a list of discovered
+    tests and ``requires()`` returning a list of all required
+    packages gathered from individual test metadata.
```

Comment:
I guess we don't have better name, but it is still 'any string that yum/dnf accepts' right? E.g. "library(foo/bar)" or "/usr/bin/nc"

Reply:
Yes, this should be basically the same as what we have in the [require](https://tmt.readthedocs.io/en/latest/spec/tests.html#require) attribute specification. I could copy & paste the same formulation here as well.

Reply:
+1

Reply:
I've added a link to the `require` spec instead.